support-frontend/assets/pages/showcase/components/hero.tsx

```diff
@@ -13,13 +14,18 @@ import HeroImg from './hero.svg';
 import { regionalContent } from './regionalContent';
 import './hero.scss';
 
-const Caption = (props: { className: string; captionText: string }) => (
-	<div className={props.className}>
-		<figcaption className="showcase-hero__caption">
-			<p>{props.captionText}</p>
-		</figcaption>
-	</div>
-);
+function Caption(props: {
+	className: string;
+	captionText: EmotionJSX.Element;
```

Comment:
Yeah I think it's fine to have this as JSX. I'd say change the type to `JSX.Element` though